src/overlords/defense/outpostDefense.ts

```diff
@@ -84,21 +85,25 @@ export class OutpostDefenseOverlord extends CombatOverlord {
 
 		const {attack, rangedAttack, heal} = this.getEnemyPotentials();
 
+		if (attack > 30 || rangedAttack > 30) {
+			// return;
+			this.wishlist(1, CombatSetups.hydralisks.boosted_T3);
+		}
 		const hydraliskSetup = mode == 'NORMAL' ? CombatSetups.hydralisks.default : CombatSetups.hydralisks.early;
 		const hydraliskAmount = this.computeNeededHydraliskAmount(hydraliskSetup, rangedAttack);
-		this.wishlist(hydraliskAmount, hydraliskSetup, {priority: this.priority - .2, reassignIdle: true});
+		this.wishlist(Math.min(hydraliskAmount, 2), hydraliskSetup, {priority: this.priority - .2, reassignIdle: true});
```

Comment:
This is to prevent insane escalations and draining of rooms for remote outposts. Ideally we should have a "give up" function instead, but I'll tag it as an improve eventually